providers/implementations/kdfs/pbkdf2.c.in

```diff
@@ -59,18 +59,17 @@ typedef struct {
     PROV_DIGEST digest;
     int lower_bound_checks;
     OSSL_FIPS_IND_DECLARE
-} KDF_PBKDF2;
+} KDF_PBKDF2_CTX;
```

Comment:
This seems like an unnecessary change that complicates reviewing.  We don't name these _CTX elsewhere that I remember, so why here?